doc/testing_guide.md

```diff
@@ -195,6 +195,12 @@ Please note that the benchmarking process is not ready for the PoC code yet and
 
 Benchmarking and updating weights should be done each time a new extrinsic is added to any pallet (weights are used to estimate transaction fees). Weights are obligatory for extrinsics that are available for users.
 
+To create a node with benchmarking capabilities you need to compile it with:
+```bash
```

Comment:
I think just a comma:  `capabilities, you need to compile`

Reply:
Done.